package.json

```diff
@@ -5,22 +5,21 @@
   "license": "Apache-2.0",
   "author": "Morgan Stanley",
   "scripts": {
-    "bootstrap": "lerna bootstrap",
-    "postinstall": "npm run bootstrap",
     "lint": "eslint . --quiet",
     "lint:fix": "eslint . --quiet --fix",
-    "clean": "lerna run clean",
-    "build": "lerna run build:lerna --stream",
+    "clean": "npm run clean --workspaces --if-present",
+    "build": "npm run build --workspaces --if-present",
     "test": "gulp test",
     "start": "gulp server",
     "docs": "typedoc",
-    "watch": "lerna run --parallel watch",
-    "deploy": "lerna run deploy"
+    "deploy": "npm run deploy --workspaces --if-present",
+    "version": "npx lerna version --no-push --no-git-tag-version"
```

Comment:
If we can replace this with something better, we could remove lerna.json and the whole dependency on it, isn't it?

Reply:
Absolutely 100%.  I had lerna config deleted originally but then after a few minutes searching I found nothing better to help with versioning so added this npm script in to help facilitate that.  I really like how it prompts if you want patch, minor, major.